frontend/src/pages/AuthRedirect.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../utils/hooks';
+
+export default function AuthRedirect() {
+  const navigate = useNavigate();
+
+  const auth = useAuth();
+
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
+
+    const accessToken = params.get('accessToken');
+    const refreshToken = params.get('refreshToken');
+
+    auth.signIn({
+      accessToken: accessToken!,
+      refreshToken: refreshToken!,
+    });
```

Comment:
Let's add some error handling here incase either token is missing

Reply:
Will be addressed in separate PR